style.css

```diff
@@ -323,11 +360,11 @@
 }
 
 .speaker-four {
-  top: 68rem;
+  top: 70rem;
 }
 
 .speaker-five {
-  top: 72rem;
+  top: 73rem;
 }
 
 .speaker-six {
```

Comment:
- [ ] some of your stylings is broken for the Home page, kindly make sure to fix it so your website looks clean and organized

|||
|-|-|
|![Screenshot_20230318_000343](https://user-images.githubusercontent.com/25227022/226066805-f7959aa2-f013-4a64-b0a3-80975072277e.png)|![Screenshot_20230318_000325](https://user-images.githubusercontent.com/25227022/226066765-65f55c30-fe2d-437b-bd70-f12360eaa6ba.png)|
|![Screenshot_20230318_000311](https://user-images.githubusercontent.com/25227022/226066747-515a91a3-9fa3-4f96-9117-057bbcd7de70.png)|![Screenshot_20230318_000253](https://user-images.githubusercontent.com/25227022/226066727-3b28e1b7-9cd6-4ad7-b9cc-6ca54d885c81.png)|